systemd/osdf-cache.yaml

```diff
@@ -24,6 +24,6 @@ XRootD:
   ## Authfile is the xrootd authorization file for VOMS/X.509 auth
   ## XXX remove this once #564 (https://github.com/PelicanPlatform/pelican/issues/564) is done
   Authfile: /run/stash-cache-auth/Authfile
-  RunLocation: /run/pelican/osdf-cache/xrootd
 Cache:
   DataLocation: "/mnt/osdf"
+  RunLocation: /run/pelican/osdf-cache/xrootd/cache
```

Comment:
Why not just `/run/pelican/rootd/cache`? Seems redundant to specify cache twice in the path